### DISCUSSION TITLE
lastUpdated same date display on cloudflare pages

### DISCUSSION THREAD

**Discussion by gkibria** [vuejs/vitepress#3580] [Q&A] [answered]
Hi,

The `lastUpdated` display shows the date correctly on locally but not on Cloudflare pages. On Cloudflare, all the date shows the same, that is the last commit date. I am using v1.0.0-rc.43. Any idea why this was happening?

BR

**Comment by brc-dd** [comment, marked as answer]
Change your `docs:build` command to `git fetch --unshallow && vitepress build docs`

**Reply by gkibria** [reply]
Thanks, it works! I assume this is a Cloudflare hack. On Netlify it works with default build command.

**Reply by brc-dd** [reply]
Yeah. Cloudflare does shallow clone which only has the latest commit information. That's why all the pages were showing that.
